android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java

```diff
@@ -387,11 +387,13 @@ static void trim(ReadableMap options, final Promise promise, ReactApplicationCon
     cmd.add("-preset");
     cmd.add("ultrafast");
     // NOTE: DO NOT CONVERT AUDIO TO SAVE TIME
+    cmd.add("-c:v");
+    cmd.add("copy");
     cmd.add("-c:a");
     cmd.add("copy");
     // NOTE: FLAG TO CONVER "AAC" AUDIO CODEC
-    cmd.add("-strict");
-    cmd.add("-2");
+    // cmd.add("-strict");
```

Comment:
Why these lines are commented?

Reply:
-strict -2 doesn't seem needed any more.

I added the video copy for performance, was having trouble with android and after adding the -c:v copy the runtime had a huge improvement